examples/pytorch/dis_kvstore/client.py

```diff
@@ -16,43 +21,45 @@
 DATA.append(th.tensor([[3.,3.,3.,],[3.,3.,3.,]]))
 DATA.append(th.tensor([[4.,4.,4.,],[4.,4.,4.,]]))
 
-edata_partition_book = {'edata':th.tensor([0,0,1,1,2,2,3,3])}
-ndata_partition_book = {'ndata':th.tensor([0,0,1,1,2,2,3,3])}
+class ArgParser(argparse.ArgumentParser):
+    def __init__(self):
+        super(ArgParser, self).__init__()
 
-def start_client():
-    time.sleep(3)
+        self.add_argument('--machine_id', type=int, default=0,
+                          help='Unique ID of each machine.')
+        self.add_argument('--ip_config', type=str, default='ip_config.txt',
+                          help='IP configuration file of kvstore.')
+        self.add_argument('--backup_count', type=int, default=2,
+                          help='Count of backup client.')
 
-    client = dgl.contrib.start_client(ip_config='ip_config.txt', 
-                                      ndata_partition_book=ndata_partition_book, 
-                                      edata_partition_book=edata_partition_book,
-                                      close_shared_mem=True)
+def start_client(args):
+    """Start kvstore service
+    """
+    server_namebook = dgl.contrib.read_ip_config(filename=args.ip_config)
 
-    tensor_edata = client.pull(name='edata', id_tensor=th.tensor([0,1,2,3,4,5,6,7]))
-    tensor_ndata = client.pull(name='ndata', id_tensor=th.tensor([0,1,2,3,4,5,6,7]))
+    my_client = KVClient(server_namebook)
 
-    print(tensor_edata)
-    client.barrier()
+    my_client.connect()
 
-    print(tensor_ndata)
-    client.barrier()
+    if my_client.get_id() % args.backup_count == 0:
+        my_client.set_partition_book(name='entity_embed', partition_book=partition)
+    else:
+        time.sleep(3)
```

Comment:
why we need to sleep 3 secends?

Reply:
The backup server need to wait the master server to create the shared-tensor. Both sleep(1) and sleep(2) are OK.

Reply:
Can we make it a check and go/fail model?